src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -638,6 +639,47 @@ def test_configure_active_load_levels(multi_instrument_session):
     assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].active_load_vcom == pytest.approx(3.0, rel=1e-3)
 
 
+def test_load_get_unload_patterns(multi_instrument_session):
+    '''Test basic pattern methods.
+
+    - load_pattern
+    - get_pattern_name
+    - unload_all_patterns
+    '''
+    test_name = 'multiple_patterns'
+    multi_instrument_session.load_pin_map(get_test_file_path(test_name, 'pin_map.pinmap'))
+
+    multi_instrument_session.load_pattern(get_test_file_path(test_name, 'pattern_a.digipat'))
+    multi_instrument_session.load_pattern(get_test_file_path(test_name, 'pattern_b.digipat'))
+
+    assert multi_instrument_session.get_pattern_name(0) == 'first_pattern'
+    assert multi_instrument_session.get_pattern_name(1) == 'second_pattern'
+
+    multi_instrument_session.unload_all_patterns(unload_keep_alive_pattern=True)
+    try:
+        multi_instrument_session.get_pattern_name(0)
+        assert False
+    except nidigital.Error as e:
+        assert e.code == -1074135025
+        assert e.description.find('Invalid parameter.') != -1
+
+
+def test_configure_pattern_burst_sites(multi_instrument_session):
+    test_name = 'multiple_patterns'
+    configure_session(multi_instrument_session, test_name)
+    multi_instrument_session.load_pattern(get_test_file_path(test_name, 'pattern_b.digipat'))
+    multi_instrument_session.start_label = 'second_pattern'
+    multi_instrument_session.selected_function = nidigital.SelectedFunction.DIGITAL
+    multi_instrument_session.wait_until_done(timeout=datetime.timedelta(seconds=5.0))
+
+    multi_instrument_session.sites[0, 2, 3].configure_pattern_burst_sites()
+
+    with multi_instrument_session.initiate() as _:
+        pass
```

Comment:
Since you're not doing anything inside the context manager, you can drop the `with`.

Reply:
Done.